applications/ConvectionDiffusionApplication/custom_strategies/strategies/explicit_runge_kutta_4_eulerian_convdiff_strategy.h

```diff
@@ -0,0 +1,385 @@
+class ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion : public ExplicitSolvingStrategyRungeKutta4<TSparseSpace, TDenseSpace>
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    // The base class definition
+    typedef ExplicitSolvingStrategyRungeKutta4<TSparseSpace, TDenseSpace> BaseType;
+
+    // The explicit builder and solver definition
+    typedef typename BaseType::ExplicitBuilderType ExplicitBuilderType;
+
+    /// The local vector definition
+    typedef typename TDenseSpace::VectorType LocalSystemVectorType;
+
+    /** Counted pointer of ClassName */
+    KRATOS_CLASS_POINTER_DEFINITION(ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /**
+     * @brief Default constructor. (with parameters)
+     * @param rModelPart The model part of the problem
+     * @param ThisParameters The configuration parameters
+     */
+    explicit ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion(
+        ModelPart &rModelPart,
+        Parameters ThisParameters)
+        : BaseType(rModelPart, ThisParameters)
+    {
+    }
+
+    /**
+     * @brief Default constructor.
+     * @param rModelPart The model part to be computed
+     * @param pExplicitBuilder The pointer to the explicit builder and solver
+     * @param MoveMeshFlag The flag to set if the mesh is moved or not
+     */
+    explicit ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion(
+        ModelPart &rModelPart,
+        typename ExplicitBuilderType::Pointer pExplicitBuilder,
+        bool MoveMeshFlag = false,
+        int RebuildLevel = 0)
+        : BaseType(rModelPart, pExplicitBuilder, MoveMeshFlag, RebuildLevel)
+    {
+    }
+
+    /**
+     * @brief Default constructor.
+     * @param rModelPart The model part to be computed
+     * @param MoveMeshFlag The flag to set if the mesh is moved or not
+     */
+    explicit ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion(
+        ModelPart &rModelPart,
+        bool MoveMeshFlag = false,
+        int RebuildLevel = 0)
+        : BaseType(rModelPart, MoveMeshFlag, RebuildLevel)
+    {
+    }
+
+    /** Copy constructor.
+     */
+    ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion(const ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion &Other) = delete;
+
+    /** Destructor.
+     */
+    virtual ~ExplicitSolvingStrategyRungeKutta4ConvectionDiffusion() = default;
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    /**
+     * @brief Initialization of variables.
+     * @details In this method, we call the base strategy initialize and initialize the projection variable.
+     * This is required to prevent OpenMP errors as the projection variable is stored in the non-historical database.
+     */
+    void Initialize() override
+    {
+        KRATOS_TRY;
+
+        auto& r_model_part = BaseType::GetModelPart();
+        auto& r_process_info = r_model_part.GetProcessInfo();
```

Comment:
```suggestion
        const auto& r_process_info = r_model_part.GetProcessInfo();
```

Reply:
Since I modify the process info, it cannot be const. Is this a problem?

Reply:
No. I didn't find where are you modifying the `ProcessInfo` in this scope.

Reply:
[Here](https://github.com/KratosMultiphysics/Kratos/pull/7272/files/d1c21f2d6f36610a0ddd148deea6b5c162728b37#diff-f2edbb58a618d0db44828cddbca9132572bdcaf53194bf8301291ab0671acd36R154). I will now put 0.0 instead of 1.0.

Reply:
Can we resolve this?